backend/development.ini.sample

```diff
@@ -490,21 +515,27 @@ caldav.radicale.headers.Access-Control-Expose-Headers = Etag
 ; search.elasticsearch.port = 9200
 # global elasticsearch timeout in seconds
 ; search.elasticsearch.request_timeout = 60
+
 # main index alias use to search/index content in elasticsearch
-; search.elasticsearch.index_alias = tracim_contents
+# search.elasticsearch.index_alias = tracim_contents
+
 # template_name for all index created currently only {index_alias} and {date} variable are usable.
 # be carefull this template is use to find all index to delete with tracimcli search delete command
 # default value is "{index_alias}-{date}"
 ; search.elasticsearch.index_pattern_template = {index_alias}-{date}
+
 # if you use ingest mode in elasticsearch, you can activate file content ingest
 # for search (default as false)
-; search.elasticsearch.use_ingest = false
+; search.elasticsearch.use_ingest = False
+
 # define mimetypes to be plain-text indexed. List of mimetypes, separated by a coma ",".
-# if set to empty, index documents of any mimetype.
-; search.elasticsearch.ingest.mimetype_whitelist = text/markdown,text/plain
+# if set to empty, index documents of any mimetype:
```

Comment:
We need better explain about empty by default, maybe something like visible in next parameter.

Reply:
fixed